### PR TITLE
Cic prod frontend

### PR DESCRIPTION
**Pull request by AubreyTharpe**
https://cm-jira.usa.gov/browse/BRMP-8663
https://cm-jira.usa.gov/browse/BRMP-8665
https://cm-jira.usa.gov/browse/BRMP-8666
https://cm-jira.usa.gov/browse/BRMP-8667
https://cm-jira.usa.gov/browse/BRMP-8668
https://cm-jira.usa.gov/browse/BRMP-8669

Changes
--
Refactored frontend to work with current drupal

### CODE REVIEW

**Comment by AubreyTharpe** [comment]
Merging to deploy code

File: src/components/features/manufacturers/offeringsService.js [hunk 1]
```diff
@@ -36,17 +37,17 @@ const parseOfferings = (data) => {
             let obj = {};
```
Comment (snoopysnake) [inline src/components/features/manufacturers/offeringsService.js:37]: should obj fields be set to an empty string `''` if field does not exist in data?

File: src/components/services/apiServices/vendorContractAPI.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+export const getBPAAssociated = (vendorId) => {
+  return axios.get(`/jsonapi/node/vendor_contract?filter[field_av_vendor_associated.id]=${vendorId}&filter[field_isbpa]=1&sort=title`).then(res => {
+      return Promise.resolve(res.data.data);
+  });
+}
+
+export const getPublishedAcquisitionVehiclesAssociated = () => {
+  return axios.get(`/jsonapi/node/vendor_contract?filter[status]=1&sort=title`).then(res => {
+      return Promise.resolve(res);
```
Comment (snoopysnake) [inline src/components/services/apiServices/vendorContractAPI.js:29]: can return `Promise.resolve(res.data)` for succinctness (same follows for `getPublishedAcquisitionVehicles` and `getPubishedBPAs`

File: src/components/services/apiServices/vendorAPI.js [hunk 3]
```diff
@@ -1,25 +1,25 @@
...
-    return axios.get(`/jsonapi/node/vendor_information_associated?filter[field_manufacturer_associated.id]=${manufacturerId}&sort=title`).then(res => {
+    return axios.get(`/jsonapi/node/vendor_information?filter[field_vendor_authorized_oem.id]=${manufacturerId}&sort=title`).then(res => {
         return Promise.resolve(res.data.data);
     });
 }
 
 export const getPublishedVendors = () => {
-    return axios.get(`/jsonapi/node/vendor_information_associated?filter[status]=1&sort=title`).then(res => {
+    return axios.get(`/jsonapi/node/vendor_information?filter[status]=1&sort=title`).then(res => {
         return Promise.resolve(res);
```
Comment (snoopysnake) [inline src/components/services/apiServices/vendorAPI.js:17]: `res.data.data`

**Pull request merged**